### ISSUE TITLE
Webpack publicPath is not set during build-javascript stage

### ISSUE THREAD

**Issue by tribou** [gatsbyjs/gatsby#392]
Is there a reason the `output.publicPath` is not being set during the `build-javascript` stage?  I found that my imported images (using `file-loader`/`url-loader`) were generating relative paths which would break when browsing away from the root path.  Adding this to `gatsby-node.js` appears to fix:

```js
  if (env === 'build-javascript') {

    config.merge({
      output: {
        publicPath: '/',
      },
    })

  }
```

I'm guessing a more permanent solution would need to respect the `linkPrefix` config.

**Comment by KyleAMathews** [comment]
I can't think of/remember any reason why `publicPath` isn't set there (and for the build-html stage)... and yeah, it probably should use the `linkPrefix` that's set. Want to create a PR?

**Comment by KyleAMathews** [comment]
Fixed by #502
